tests/samples_smoke_tests/common/samples_common_test_clas.py

```diff
@@ -0,0 +1,432 @@
+            elif ('l' == k and 'pascal_voc_classes' in param['l']):
+                param['l'] = os.path.join(Environment.env['test_data'], param['l'])
+            elif ('pp' == k):
+                param['pp'] = gpu_lib_path
+            elif ('r' == k):
+                if len(param['r']) > 0:
+                    param['r'] = os.path.join(Environment.env['test_data'], param['r'])
+            elif ('o' == k):
+                param['o'] = os.path.join(Environment.env['out_directory'], param['o'])
+            elif ('wg' == k):
+                param['wg'] = os.path.join(Environment.env['out_directory'], param['wg'])
+            elif ('we' == k):
+                param['we'] = os.path.join(Environment.env['out_directory'], param['we'])
+            elif ('fg' == k):
+                param['fg'] = os.path.join(Environment.env['test_data'], param['fg'])
+            elif ('labels' == k):
+                label_folder = os.path.dirname(executable_path.split(' ')[-1])
+                param['labels'] = os.path.join(label_folder, param['labels'])
+            elif ('lb') == k:
+                param['lb'] = os.path.join(Environment.env['test_data'], param['lb'])
+
+    @staticmethod
+    def get_cmd_line(param, use_preffix=True, long_hyphen=[]):
+        line = ''
+        for key in sorted(param.keys()):
+            if use_preffix and any([x for x in long_hyphen if key == x]):
+                line += '--{} {} '.format(key, param[key])
+            elif use_preffix and key not in long_hyphen:
+                line += '-{} {} '.format(key, param[key])
+            elif not use_preffix:
+                line += '{} '.format(param[key])
+        return line
+
+    @staticmethod
+    def check_is_perf(stdout):
+        # This function check if FPS in stdout. If yes - then need to run this sample for perfomance
+        for line in stdout:
+            if 'fps' in line.lower():
+                return True
+        return False
+
+    @staticmethod
+    def check_has_niter(param):
+        # Check if niter has already in params, so it was set before
+        if 'niter' in param:
+            return True
+        return False
+
+    @staticmethod
+    def find_fps(retcode, stdout, stderr):
+        stdout = stdout.split('\n')
+        for line in stdout:
+            if 'fps' in line.lower():
+                return float(re.findall("\d+\.\d+", line)[0])
+
+    @staticmethod
+    def write_csv(sample_name, sample_type, cmd_perf, fps_perf):
+        csv_path = Environment.env['perf_csv_name']
+        with open(csv_path, 'a', newline='') as f:
+            perf_writer = csv.writer(f, delimiter='|', quotechar='|', quoting=csv.QUOTE_MINIMAL)
+            perf_writer.writerow([sample_name, sample_type, cmd_perf.rstrip(), fps_perf])
+
+    @staticmethod
+    def get_empty_cmd_line(param, use_preffix=True, long_hyphen=[]):
+        line = ''
+        return line
+
+    @staticmethod
+    def get_hello_cmd_line(param, use_preffix=True, long_hyphen=[]):
```

Comment:
if variable (`long_hyphen`) is used somewhere in another function, but required to be specified in other functions APIs, use `**kwargs` for such purposes